test/blob_test.js

```diff
@@ -1,4 +1,4 @@
-suite.only("Azure Blob", function() {
+suite("Azure Blob", function() {
```

Comment:
I noticed that you're removing `.only` but not moving the sleeps.  Does that mean the sleeps fixed the issue?